crt_portal/cts_forms/tests/test_forms.py

```diff
@@ -361,6 +361,12 @@ def setUp(self):
         for index, report in enumerate(reports):
             report.assigned_section = sections[index % len(sections)]
             report.save()
+        # generate random reports associated with a different email address
+        reports = [Report.objects.create(**SAMPLE_REPORT) for _ in range(5)]
+        for report in reports:
+            report.assigned_section = 'VOT'
+            report.contact_email = 'SomeoneElse@usa.gov'
+            report.save()
```

Comment:
It looks like the new test here is the only one using these reports -- let's move their generation into that test to avoid creating these for every `FormNavigationTests` test.